pilot/pkg/networking/core/v1alpha3/networkfilter.go

```diff
@@ -32,22 +32,25 @@ import (
 	"istio.io/istio/pilot/pkg/networking/util"
 )
 
-// buildInboundNetworkFilters generates a TCP proxy network filter on the inbound path
-func buildInboundNetworkFilters(instance *model.ServiceInstance) []listener.Filter {
-	clusterName := model.BuildSubsetKey(model.TrafficDirectionInbound, "",
-		instance.Service.Hostname, instance.Endpoint.ServicePort)
+// buildInboundNetworkFilter generates a TCP proxy network filter on the inbound path
+func buildInboundNetworkFilter(instance *model.ServiceInstance) listener.Filter {
```

Comment:
i think this name is too general for what the function actually does.

Reply:
The name is exactly what this function does: it creates an inbound network filter. Nothing else is an inbound network filter; they could be an inboundMongoFilter, or an inboundHTTPFilter, or whatever, but this is the baseline network filter.

Reply:
You shouldn't need to modify this at all. As stated earlier, you need to use buildOutboundNetworkFilter